float64 input
float64 setpoint
float64 output
float64 error
float64 integral_error
float64 diff_error
float64 windup_limit
float64 output_p
float64 output_i
float64 output_d
#output = output_p + output_i + output_d
